lib/smart_answer_flows/inherits-someone-dies-without-will.rb

```diff
@@ -395,13 +357,15 @@ def define
         option :yes
         option :no
 
-        save_input_as :great_aunts_or_uncles
+        on_response do |response|
+          calculator.great_aunts_or_uncles = response
+        end
 
-        next_node do |response|
-          case response
-          when "yes"
+        next_node do
+          if calculator.great_aunts_or_uncles?
             outcome :outcome_45
-          when "no"
+          else
+            calculator.next_steps = [:ownerless_link]
```

Comment:
I think this is a constant, tied specifically to that particular outcome page - we should probably set it there. Otherwise we need to make sure, every time we route to outcome_46 we need to set next_steps.

(Same for outcome_25 and outcome_67)